src/components/popup/tx-item.tsx

```diff
@@ -126,8 +126,12 @@ export const TxItem: React.FC<TxItemProps & BoxProps> = ({ transaction, ...rest
         <TxItemIcon transaction={transaction} />
         <Stack flexGrow={1} spacing="base-tight">
           <SpaceBetween>
-            <Title as="h3">{getTxTitle(transaction as any)}</Title>
-            <Title as="h3">{getTxValue(transaction, isOriginator)}</Title>
+            <Title as="h3" fontWeight="normal">
```

Comment:
I would suggest changing it here
https://github.com/blockstack/stacks-wallet-web/blob/3c87a0403094c9c157497d510af21db301207303/src/components/typography.tsx#L106

Reply:
Actually all the titles should always be fontWeight=500 see, that's why I added specifically here.
See https://github.com/blockstack/stacks-wallet-web/issues/1261